example/full-config.php

```diff
@@ -77,14 +73,9 @@
                 'configuration' => 'orm_default', // Actually defaults to the entity manager config key, not hard-coded
             ],
         ],
-        'event_manager' => [
-            'orm_default' => [
-                'subscribers' => [],
```

Comment:
Doctrine ORM v.3 provides [subscribing to events](https://www.doctrine-project.org/projects/doctrine-orm/en/3.0/reference/events.html#listening-and-subscribing-to-lifecycle-events). Why are you going to drop support of the subscribers from the container?

Reply:
You are right, I've been deceived by the removal of `EventManager` support from the `Connection`.
I've restored it, and moved the key usage from `doctrine.connection` to `doctrine.entity_manager`.